docs/core/install/dependencies.md

```diff
@@ -241,16 +241,17 @@ Based on your linux distribution, you may need to install additional dependencie
 
 Ubuntu distributions require the following libraries to be installed:
 
-- liblttng-ust0
-- libcurl3 (for 14.x and 16.x)
-- libcurl4 (for 18.x)
-- libssl1.0.0
-- libkrb5-3
-- zlib1g
+- libc6
```

Comment:
Where did you get the requirements for libc6 and libgcc1? I have not seen a need to install these independently for .NET scenarios.

Reply:
From here:
* https://github.com/dotnet/runtime/blob/a755e742cad93fb173b47a95f68168c29ae95fb6/src/installer/pkg/packaging/deb/dotnet-runtime-deps-debian_config.json#L32
* https://github.com/dotnet/dotnet-docker/blob/75e441fdb26bdffa5f58437cd77048586016ec96/src/runtime-deps/3.1/bionic/amd64/Dockerfile#L8

This was before my time so I don't know the history.  Perhaps @dagood or @MichaelSimons has more context.

Reply:
As far as Git can tell me, these originated in https://github.com/dotnet/core-setup/pull/3317.